### PR TITLE
Separate task ajax requests into angular service.

### PR DESCRIPTION
**Pull request by lydiagu**

### CODE REVIEW

File: orchestra/static/orchestra/common/js/orchestra.services.js [hunk 1]
```diff
@@ -67,3 +67,33 @@ serviceModule.factory('orchestraService', function() {
   };
   return orchestraService;
 });
+
+serviceModule.factory('orchestraTasks', function($http) {
+
+  var newTask = function(taskType) {
```
Comment (jrbotros) [inline orchestra/static/orchestra/common/js/orchestra.services.js:73]: At some point we should probably have an API service so we have all our URLs in one place, but totally good for now!

File: orchestra/static/orchestra/common/js/orchestra.services.js [hunk 2]
```diff
@@ -67,3 +67,33 @@ serviceModule.factory('orchestraService', function() {
...
+    for (var taskType in tasks) {
+      if (taskType != 'complete') {
+        numTasks += tasks[taskType].length;
+      }
+    }
+    return numTasks;
+  };
+
+  var service = {
+    newTask: newTask,
```
Comment (jrbotros) [inline orchestra/static/orchestra/common/js/orchestra.services.js:93]: Any reason you're not defining these functions inline? Usually our style but I don't have strong feelings.

**Comment by jrbotros** [comment]
lgtm!

File: orchestra/static/orchestra/common/js/orchestra.services.js (hunk 2 above)
Comment (lydiagu) [inline orchestra/static/orchestra/common/js/orchestra.services.js:93]: More readable in general. I was copying the high level structure in https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/static/orchestra/common/js/orchestra.services.js and just now noticed that the nested functions are inline.

File: orchestra/static/orchestra/common/js/orchestra.services.js (hunk 1 above)
Comment (lydiagu) [inline orchestra/static/orchestra/common/js/orchestra.services.js:73]: Yes, agreed!

**Pull request merged**